starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,610 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  } else if (number2 > number1) {
+    return number2;
+  }
+  return number1;
```

Comment:
Hey Nora. Here you could have used the Math.max(number1, number2) -> it automatically returns the highest number between them.